package.json

```diff
@@ -116,7 +116,7 @@
     "parse-server": "bin/parse-server"
   },
   "optionalDependencies": {
-    "@node-rs/bcrypt": "0.4.1"
+    "@node-rs/bcrypt": "^1.1.0"
```

Comment:
Can you please remove the range operator.

@davimacedo Aren't optional dependencies updated automatically by Snyk?

Reply:
Resolved